docs/conf.py

```diff
@@ -0,0 +1,106 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# Problems with imports? Could try `export PYTHONPATH=$PYTHONPATH:`pwd`` from root project dir...
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))  # Source code dir relative to this file
+
+# -- Project information -----------------------------------------------------
+
+project = 'LUSID Python Preview SDK'
+author = 'FINBOURNE Technology'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'
```

Comment:
Is this the release version for these sphinx docs? If it is meant to be the version of the sdk then it will need to be updated and we will have to work out how to make sure this value is bumped with the overall version increments.

Reply:
Yes, version of the docs rather than the SDK. Version of the SDK gets published on the front page:
![image](https://user-images.githubusercontent.com/51993708/117972893-9b8f0200-b323-11eb-8168-a8e4af26ced2.png)